src/news.js

```diff
@@ -0,0 +1,50 @@
+class News {
+  constructor() {
+    this.date = this.getDate()
+  }
+  getDate() {
+    let today = new Date();
+    let day = String(today.getDate()).padStart(2, '0');
+    let month = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
+    let year = today.getFullYear();
+    today = `${year}-${month}-${day}`
+    return today
+  }
+  connectAPI() {
+    let url = "https://content.guardianapis.com/search?"
+    let tag = "&tag=politics/politics"
+    let date = this.getDate()
+    let api = "&api-key=be4ccfea-6ee5-4d67-aa1e-2d8c1a87574e"
+    fetch(`${url}${tag}&from-date=${date}${api}&show-fields=all`)
+      .then(res => {
+        return res.json()
+      })
+      .then(data => {
+        let res = data.response.results
+        this.renderNews(res)
+      })
+  }
+  renderNews(res) {
```

Comment:
You might consider separating the processing of the API results from the creation of DOM elements -- it's hard to write a meaningful test for what this function does otherwise. The recording of Alice's live coding of the Notes app might give some hints for how to do this.